WORKSPACE

```diff
@@ -19,10 +19,14 @@ local_repository(
 
 http_archive(
     name = "bazel_skylib",
-    sha256 = "eb5c57e4c12e68c0c20bc774bfbc60a568e800d025557bc4ea022c6479acc867",
-    strip_prefix = "bazel-skylib-0.6.0",
-    url = "https://github.com/bazelbuild/bazel-skylib/archive/0.6.0.tar.gz",
+    urls = [
```

Comment:
Is this hunk missing from #261?

Reply:
Strictly speaking nope, protobuf builds with skylib 0.6.0, but unittest.bzl file that I use  in this PR was polished later than that. I can move skylib to //rust:repositories.bzl if you prefer that?